iOS/Flows/Conversation/SwipeableInputAccessoryMessageSender.swift

```diff
@@ -0,0 +1,212 @@
+//
+//  SwipeableInputAccessoryView.swift
+//  Jibber
+//
+//  Created by Martin Young on 12/6/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+protocol MessageSendingViewControllerType: UIViewController {
+    func set(shouldLayoutForDropZone: Bool)
+    func getCurrentMessageSequence() -> MessageSequence?
+    func set(messageSequencePreparingToSend: MessageSequence?, reloadData: Bool)
+    func sendMessage(_ message: Sendable)
+    func createNewConversation(_ sendable: Sendable)
+}
+
+protocol MessageSendingCollectionViewType: CollectionView {
+    func getMessageDropZoneFrame(convertedTo view: UIView) -> CGRect
+    func getDropZoneColor() -> Color?
+    func getNewConversationContentOffset() -> CGPoint
+}
```

Comment:
Any VC that wants to use this delegate object needs to conform to these protocols.